recipes/msgpack-c/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "msgpack-c" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "eb20b4bf15f20bad149ec82fffac74f16de2a8a797e321a3f8189d63263a511e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: cpp-{{ version }}.tar.gz
```

Comment:
This doesn't have to match the filename in the URL. So it could be `{{ name }}-{{ version }}.tar.gz`, which is handy when you are building locally and trying to figure out what is in your source cache. 😆